modules/common/react/lib/styles/hideMouseFocus.ts

```diff
@@ -5,9 +5,9 @@ import {CSSObject} from '@emotion/core';
  * Requires wrapping your components in an InputProvider component.
  */
 export const hideMouseFocus: CSSObject = {
-  [`[data-whatinput='mouse'] &:focus,
-    [data-whatinput='touch'] &:focus,
-    [data-whatinput='pointer'] &:focus`]: {
+  [`[data-whatinput="mouse"] &:focus,
```

Comment:
Can we back out of the formatting changes here and in `styled.ts` below?

Reply:
These aren't formatting - this was intentional. I ran into issues with the `StaticStates` wrapper because it the regex was looking for `"` and not `'`. 

I updated this to be more technically correct (attributes generally use `"`), and updated the regex in `styled` to catch either